packages/core/src/utils/transformers.ts

```diff
@@ -45,6 +46,49 @@ export function RecordTransformer<T>(Class: { new (...args: any[]): T }) {
   })
 }
 
+export function CredentialDefinitionTransformer() {
```

Comment:
Maybe we can place these next to the record classes so they're bundled with the correct module